### ISSUE TITLE
Ideas to help

### ISSUE THREAD

**Issue by AlistairB** [scientific-defense-force/this-is-the-world-site#1]
https://thisistheworld.net describes a problem. What are some solutions to this problem?

Personally I believe complete rethink of society is desirable and feasible.

I think we need to work on

- Clearly communicate the current issues to the educated middle class of the world.
- Define a viable alternative to the current system.

https://thisistheworld.net and https://opensocialism.com are my respective initial attempts at this.

I'm very interested in discussion on this or alternative strategies.

**Comment by niltwill** [comment]
You might wanna take a look at Murray Bookchin's [Communalism](https://en.wikipedia.org/wiki/Communalism_(political_philosophy)). It's a nice summary of an ideal system, IMHO. I would like to see how far such ideas could go in real life, if that's at all possible.

There's also [this proposition](http://spoirier.lautre.net/en/infoliberalism.htm), but relying on the Internet is not a good idea, since not everyone may have access to it, or the electric grid may not remain forever either.

Now...People need an easy-to-use option. One that average people and little kids and old people, anyone ..... can understand and participate in with ease - not just the intellectual bigshots. If you cannot explain your system to a 3 or 5-year old kid, then that system is no good. Same thing with the economy or monetary system. Anyone should be able to participate in it, regardless of age, skill, talents, social standing, makings.

The argument asks "to what extent should common people participate in the OPEN system/governance?", to which the obvious answer is: "it depends on their abilities, to whatever they can contribute with." There is the concept of [meritocracy](https://en.wikipedia.org/wiki/Meritocracy), but it's not without flaws.

Let's assume that "merit" translates to "skill that one can apply practically". Now, how do you measure technical competence? As Wikipedia points out, "the reliability of the authority and system that assesses each individual's merit is another point of concern."

Let's consider Murray Bookchin's Communalism again, where social hierarchy would be removed, and a general class - based on the particular community's needs - would be molded (that is, people would no longer be classified by their professions, such as a laborer, engineer, etc. but rather each one would have the same classification as a general participant of humanity, being able to try his luck at whatever he'd like [including free education], we might have a preliminary test to show if he is really capable for a particular work or job - instead of the limited education [one, that forces graduation] and work opportunities of today...mentors could be a thing to help them as well, and so on.)

I know that sounds too good to be true, but I had that envisioned in my mind before, where people no longer bicker over their jobs, education results, have to be job-hunting (since there's always something available to do for them outside) or taunt others due to hierarchial differences, because they have changed from their infancy, and now pursue and participate in the general interests of their community.

Wanna work? Well, there'd be many things to do! Yeah, the community would need to do lots of things, and one could just try applying as a volunteer at whatever he fancies. Sounds too good to be true? Well, what stops people from living in such a 'system'?

(I'm not even sure if we would even have to have money in that sort of system, where instead one's trust or reputation level with their approved skills/merits could become the new 'currency'.)

Open, transparent opportunities is the key. Specialization is not bad, but people should be able to learn in different directions as well after they have settled in a job or a field, or wish to reconsider their options and explore different areas of interest, becoming generalists.

Jacque Fresco said, while talking about the Resource Based Economy: "you really need technical competence in order to arrive at decisions that make sense. You cannot arrive at decisions that makes sense by consensus. By asking people what they want. You have to find out what the area has to offer. And that's what you can determine the future by."

"The decisions are not made by the majority of the people. They are made by the majority of the people that have technical competence, that have information in the areas you wish to excel in and methods of scientific scales of performance. If you have a millions sincere people that have no technical competence, I can assure you, nothing can be accomplished."

"If you really wish to put an end to war, poverty, hunger, territorial disputes, **you must utilize all the world's resources as the common heritage of all the world's people**. Anything less than that will remain with the same problems that you've had continuously for centuries."

That's the key point, that a certain group cannot take or keep away the most resources from the others for themselves. If you have most of the pie, then other countries will want a piece from it as well and that's how you get war. Instead, if it's all shared or distributed (more) equally among everyone, no people would have to suffer in need and thus the incentive for violence would dissipate.

We live in an economy that actually wastes edible food because people lack the capital resources (money) to be able to fetch it or companies do not recycle the overproduction, instead throw it out or pitch away, instead of giving the excess to the needy. I don't understand why corporations are stuck in this mindset that if they gave away to charity, it devalues the product. Why? In an ideal world, we could feed all those hungry mouths or close to it with the right practices and legislation.

Why is it that empty houses and apartment units outnumber the homeless population? China also has notorious 'ghost towns'. That's what happens under capitalism. Houses produced for profit, not to house people. Shoes produced for profit, not to put shoes on feet. Food produced for profit, not to feed people. Everything we do involves trading these worthless pieces of paper, instead of doing things to satisfy needs. This is not how an intelligent species ought to behave.

The idea is that only the people who have technical competence (the skill to do so) will be able to direct the system, while the majority of the people can still vote on the major or fundamental changes that they propose, or perhaps talk about what they like or dislike about it.

This would be similar to open-source communities and software: anyone can contribute from the world as long as they have the competence to participate in it.

To change the future, one should first change education. Where it all starts. Perhaps the oldies will not be able to adapt to a radical, new system, but the children may be more capable or flexible for new ideas.

So the general motive is for people to thrive for achievements, results or contributions, and not money/status/fame/social position. The point is to enable everyone to participate in any area of their competence. The measure of success would be based on the fulfillment of one's individual pursuits, rather than the acquisition of wealth, property and power. We survive by the simple currency of giving. The economic system should be based on voluntary interaction, but markets may have to exist for non-voluntary interaction. In an optimal society, everyone gives back the best of their abilities, and receive everything that they need by their given work and required natural necessities to exist. The potential of an individual must not be wasted by systematic limitations, prejudices or any of the old, traditional regimes.

Also see: [Economic Democracy](https://en.wikipedia.org/wiki/Economic_democracy), nice stuff there. So how would a council operate (such as in Communalism)? "It is a form of political and economic organization in which a single place of work or enterprise, such as a factory, school, or farm, is controlled collectively by the workers of that workplace, through the core principle of temporary and instantly revocable delegates. In a system with temporary and instantly revocable delegates, workers decided on what their agenda is and what their needs are. They also mandate a temporary delegate to divulge and pursue them. The temporary delegates are elected among the workers themselves, can be instantly revoked if they betray their mandate, and are supposed to change frequently. There are no managers and all decision power and organization is based on the delegates system."

"On a larger scale, a group of delegates may in turn elect a delegate in a higher position to pursue their mandate, and so on, until the top delegates are running the industrial system of a state. In such a system, decision power rises from bottom to top from the agendas of the workers themselves, and there is not a decision imposition from the top, as would happen in the case of a power seizure by a bureaucratic layer who are immune to instant revocation."

"Councils operate on the principle of recallable delegates. This means that elected delegates may be recalled at any time through a vote in a form of impeachment."

"Workers' councils combine to elect higher bodies for coordinating between one another. This means that the upper councils are not superior to the lower councils, but are instead built from and operated by them. The national council would therefore have delegates from every city in the country. Their nature means that workers' councils do away with traditional centralized governments and instead give power indirectly to the people. This type of democratic order is called council democracy."

Ideally, a large and interdependent network filled with various interested and technical-minded people from all over the globe, grouped into extremely branched guilds/councils that would be the central management, voluntarily driven. Basically, what the Internet offers in various forums, web sites, blogs, etc., just in a global scale, in real life, real societies.

([Worker cooperatives](https://en.wikipedia.org/wiki/Worker_cooperative) sounds interesting as well.)

Besides that...Agalmic economy can help, to a certain extent. What's the point of this? Well, think a little. Instead of donating some money to the more needy, most people would selfishly buy something they don't really need at all, such as a new shirt, accessory or similar. So I assume that many of us may have lots of 'crap' lying around that somebody else would want. One man's trash is another's treasure. And this is where agalmics comes into play!

Agalmics, in short: what exists and is unwanted is given away freely. Problems only arise when there is not enough of something to go around, i.e. it is scarce (such as the faculty of a person, i.e. an expert). To understand the idea behind agalmics; for example, in our current capitalism, imagine a warehouse of cookies that's going to be wasted. Many companies may have excess that they can't distribute, because people lack money to get them. Therefore, they could distribute the excess they produce (what is unwanted and should not take/waste space) freely between people, similarly to how benevolent donations of non-profit organizations work. This would also greatly improve our recycling efficiency.

The problem is that it is only viable for non-scarce goods, only if the goods are abundant. It assumes that production and automation can turn out more goods and services, resources.

Also [Range Voting](https://www.youtube.com/watch?v=e3GFG0sXIig) (see: [rangevoting.org](http://rangevoting.org/) for more information) could be a better mechanism for electing people or leaders.

**Comment by AlistairB** [comment]
@niltwill Wow, thanks for all that info and links! I hadn't heard of most of these. I have some reading to do..

I like a lot of what I see about Communalism. However, it does take one of the main ideas of Communism `From each according to his ability, to each according to his needs` which I'm not a huge fan of. I do believe in very moderate inequality to incentivize individuals to strive.

The bottom up idea of electing leaders is very interesting. I hadn't thought about this so much hmm. It reminds me of Valve Software where employees are 100% self directed and naturally form teams around what they think is the most valuable for the company to do (and leadership will emerge in this process) - http://www.valvesoftware.com/company/Valve_Handbook_LowRes.pdf

My general idea of [Open Socialism](https://opensocialism.com) is a top down directed society, which I think on paper has the potential to be the most efficient and effective. The big danger with this style is corruption and various measures are taken to limit this, primarily with all info and deliberation being open much like open source. If you have some time I'd be interested in any thoughts you might have on it.

**Comment by AlistairB** [comment]
Thanks for your thoughts :)

I think the goal with a system should be not to require humans to not be selfish, nor select particular humans with altruistic traits. It should be to align normal selfish human behaviour with the greater good.

I think the tribal system worked this way in that to get ahead and attract a good mate you could only do this by being highly valuable to the tribe (a great hunter for example). You also had to maintain a position in that community so couldn't be completely devious about how your promoted yourself.

The unicorn system is this concept applied to a whole society. I personally do think this is possible given we have incredible tools of technology and science. Of course this side steps the issue of how you might migrate from what we have now to such a system, but I think the first step is to define the template, then worry about how to get there.

**Comment by niltwill** [comment]
Then perhaps, a [reputation-based society](http://ilja-schmelzer.de/network/) or something along those lines? It may have some correlation with the so-called [honor system](https://en.wikipedia.org/wiki/Honor_system). Also related: [reputation system](https://en.wikipedia.org/wiki/Reputation_system)

We can already see the influence of data based reputation systems in real life. One needs to look no further than eBay to see that high reputation results in greater financial gain. Now if the only incentive would not only be money/profit...what heights could humanity reach then?

So then...let me try to explore more on this.

Have you ever heard about [demurrage](https://en.wikipedia.org/wiki/Demurrage#Currency) before? This is Gesell's concept. So, what is demurrage?

> A demurrage is a negative interest, a tax on holding money. This strongly discourages hoarding cash. This is a hard concept for many people to get their heads around. But the fact is that hoarding the means of exchange has negative implications for others using the money. Hoarding cash hinders circulation and this is a fundamental monetary problem. Saving should be done through assets, not cash.
> 
> Money is not wealth, it's a means of exchange. It derives its value from the agreement to use it as such. The 'store of value' function is a result of the agreement, but the agreement does not have the purpose of creating a store of value. Demurrage is based on the idea that money should perish as does produce. It is because money does not perish as quickly as produce (and other products), it is used to store wealth. But using it to store wealth destroys its use as a means of exchange, which needs unhindered circulation. In older times, farmers stored their harvest at central warehouses. They got a receipt in exchange. This receipt was used to pay others. But because the receipt represented decaying goods, it lost value over time.
> 
> The great boon of demurrage is that the negative interest is a clear incentive for those holding money to lend it out interest-free. Better still, it would greatly encourage paying in advance. This is even better, because it diminishes the need for debt. Many investments would become viable without any debt at all, simply because consumers would pay up front for much of the production of what they need. It is important to understand that the cost of society to the demurrage (negative interest) is a tiny fraction of the cost of usury. The reason for this is that the money supply is very small, because it circulates very quickly, up to a hundred times faster than usurious units. As a result, the demurrage is typically not even enough to finance the operation of the monetary system, let alone as a tool of plunder.
>
> One of the ways I could imagine demurrage in our current system would be in the form of social credit issued directly to every person to a special bank account or "Grundeinkommen", as they call it in Germany and Austria. You would not be able to transfer it to savings, but could only spend it or get a monthly tax deducted from the account. As soon as it is spent, it works as a conventional currency. Calculation done by the Wissensmanufaktur in Germany go that 1000 Euro per person per month could be for example issued directly in Germany without diluting the value of the currency. Remember also, that in that system availability of credit relative to income would be ridiculously easy. Even the alcoholic bum could buy his very modest house or apartment just by the social credit issued.
> 
> But it never occured to me that the entire currency should be encompassed by demurrage. That would bring numerous problems up and enormous avoidance issues by the population. Also cash would then have to go completely in order to make it feasible. My idea would be to combine it with social credit and instead of the state issuing money for their expense interest-free, it could issue the money by transferring a given amount to the population to be de-facto issued by them when they spend it.
> 
> And that amount – let's say 1000 Euro, 1000 $ monthly could easily be within demurrage "enticements" on a special electronic-only account. People would need to spend it or it would get taxed. You could not transfer it from that account, except via direct expenditure. For most people, it would be only a welcome additional income, except for those fallen on hard times. Likely, the only required taxes for government could be VAT-taxes and international-trade-tariffs apart from the high demurrage from the funds not spent. All other forms of money – from all kinds of income could be taxed at 0% – oh, and corporations should also pay a form of VAT. Combine it with interest free JAK-banks, interest-free mortgages, transparent money volume control and sound credit availability and you may have a good system with demurrage-issuance of government money directly into the hands of the people tied to issuance through expenditure.
> 
> As far as silver and gold are concerned – as personal investments anything can be good given time. However any currency-ties to any resources that are easily controlled by the 0,1% is utter foolishness. I can conclude that most interest-free systems would be one hell of a lot better than ours now, and they also have another thing in common: they are all mind-bogglingly unlikely, but exciting to explore upon.

Here's a model that explores social credit with demurrage: 
[scd-model.zip](https://github.com/scientific-defense-force/this-is-the-world-site/files/1281716/scd-model.zip)

Might share a bit more about what I have learned of Mutual Credit and Social Credit later.

...

As you are aware, in current western societies, humans are not so closely intertwined with each other as in a tribal system. Indeed, [this plague](https://www.theguardian.com/society/2013/jan/22/the-loneliness-epidemic) [wreaks havoc](https://www.theguardian.com/commentisfree/2016/oct/12/neoliberalism-creating-loneliness-wrenching-society-apart).

I feel it would be lovely in trying to establish a local volunteer group who would like to go, visit and help out the lonely elders or widows/widowers in the area. They could then do some chores for them, but more importantly, this would connect the people more and help them establish social relationships. Presently, what happens when you live alone and one day die alone in your home? Who would come to investigate? For how long would you rot in your home, left unattended, unknown? Sad, sad state of affairs. If the dwelling seems unmaintained for a long time, that would be another task for the volunteer group to check in and confirm this thing, and with aching heart dispose the deceased.

Also, elders should not be so avoided and displaced far from the family or communal unit, they should teach and motivate the children and thus play a more active role in societies. Adult parents may not be wise or capable enough for the task to raise children (alone). Adult parents may also not be around the children as much as required/needed for whatever reasons. Elderly supervision could make sure less would stray off the good/positive path, instilling moral values and memorable lessons in the children.

...

"By 2050, another two billion people will pile in, leading one US anthropologist, Jason Vargo, to suggest a new name for our species: **Metro sapiens**." [[source]](https://www.theguardian.com/commentisfree/2017/mar/13/warning-living-city-seriously-damage-health)

Glad I don't live [in a city](http://www.businessinsider.com/ways-city-life-is-bad-for-you-2015-7/).

"People are less caring in large groups."

Bingo. Don't people [who live in cities] understand that living on top of each other is a bad idea? One day they may realize that the key is to live in smaller clusters all across the land. To stop the cities from sucking up all of the countryside's veins and workforce. Their population density is bad enough as it is. How does your psyche feels when you are aware that there are millions of unknown people around you?

There should be a growth limit to both rural and urban population density. It could also be wise to limit or legitimize certain architectural designs for sustainability, in order to avoid unhealthy frameworks such as apartments, stupid ads and neons (stupid lights), and one that would prefer more futuristic, 'greener', healthier constructions. Can only hope that one day humans will reach a MATURE stage where they do manage to revamp the cities and towns to .... some more sustainable design. One can dream. Perhaps not in this timeline, if parallel universes are a thing, and if, from nature's perspective, we have already baked in our own inevitable extinction, but hope is a nice thing, yeah?

It seems that in our society, the role is that the cities are where people are supposed to live, and the countryside is where food is supposed to be produced. In my opinion, less people should be congregated into cities, and more should move out to the countryside to populate it up a little bit more (and therefore we would have a few more facilities here in the countryside too, more opportunities).

...

And finally, a little rant about power:

> Here's the thing about absolute power, whether it's given to one person, two people, a small group of people, or even to a large group of people: Since they aren't the entirety of the people their decisions will effect, they will often inevitably use that power for their own interests (or for the interests of those who gain influence over them), instead of for the benefit of those they have power over. Or they may even do what they think is best for the state, but be mistaken.
> 
> If you had a perfect leader or group of people that was both incorruptible and incapable of making mistakes (even a person trying to do good things can make mistakes), then giving that leader or group absolute power (assuming that even with power they are incorruptible) would work wonderfully, as they would have the power to do things for the benefit of all with nothing to stop them from doing so. (The president of the United States could not use a small portion of the military's funding to feed the homeless, for example; because he does not have the power to do this on his own, and congress would certainly not allow it, whereas a monarch would be able to mandate such.)
> 
> The common criticism for democracy is that those who are voting don't know what the hell they're talking about and they are voting for what would benefit themselves more than the whole. This is why parliamentary systems limit direct democracies.
> 
> When it comes to power, everyone can be corrupted. Everyone has a weak spot, it doesn't even have to be money, there'd be many other ways forcing or coercing people one way or another.
> 
> So is giving absolute powers to the right person the key? And how does one judge who's the right person and who isn't? No one. Of course, there could be elections, but remember how WWII started? With people electing the wrong people. And removing corruption for good is not only near impossible, it is absolutely impossible.
> 
> Essentially, there is only one way to fragment corruption (in a society) as much as possible: power is best split between as many people as functionally possible.
> 
> However, the reason most "democratic" governments are corrupt is that they aren't democratic at all. Most democracies are a sham, because they are really republics. It is thought of as a way of making democracy more manageable with a higher population, but in the end, it simply means that the people cannot make any decisions, and the people who do are a small, exclusive group of people that are entirely wealthy.
> 
> Granted, in current capitalism, this could ... or might... be fixed if everyone would actually vote for their congressmen and choose ones that are not corrupt instead of most not voting, and the small percentage that does, voting for whoever's campaign is funded the best (corporations with hands in politicians' pockets will essentially put the politicians they want into congress by funding their campaign and smearing their opponents).
> 
> The integrity of every generation of democratic government will usually be based on the integrity of the generation before it; once it is corrupt, it is very difficult to change.
> 
> Many people may vote under bribing during election. Some of them doesn't even know what they're doing: they don't know the true value of voting and its effect for years to come. There's a secret weapon during election: bribe. And worst part is that good people that actually better in taking the lead tend to get wiped out by bad people.
> 
> Corruption happens depends on how the generation acts during youth (and their parents), the knowledge of the citizens in the present about elections, their (citizens) immunity to bribes, the election campaigns used and the integrity of chosen leader. In capitalism, at least...

**Comment by niltwill** [comment]
Lastly, I have this one rant left to share, which might help or at least inspire a few more ideas and thoughts for further discussion elsewhere.

Capitalism is notorious for its overproduction crises. You have to produce to be able to sustain yourself ("grow or die"), but the big problem happens when you cannot manage to sell what you product. Self-restraint is lacking, despite how technological advancement or development could be leveraged to work less, and then to enjoy life with more abundant free time.

If they can no longer sell their product, then they post an advertisement, and hire a sales manager and so on, to keep/rise a share of the market. So companies are going/plotting against each other more intensively, and shove down the - in many cases - superfluous trash on consumers' throats. But development or progress should serve/increase our free time, rather than this never-ending battle against each other.

The market cannot regulate this, because from its very logic derives that progress results in an accelerating economy, despite how there is no need for that at all. There is no need for it, right at the moment when the concurrency would like to breathe up and relax. But we know that in capitalism, striving or straining is a pressure or coercion, otherwise the company is swept away from the market. Which means...bankruptcy.

Struggling is mandatory, or else you'll have no money to sustain yourself in the system. Satiating the basic necessities is already solved from the production-technological aspect (just understand that we have much that goes to waste unnecessarily), the problem lies in distribution.

You could mend some of this distribution by using basic income. Though many are afraid that then the market would not twirl; no profit production, GDP expansion. Such is capitalism. But even with basic income, people would have/retain their calling in life; they would not sit lazily on their asses at home as many like to believe or imagine.

Yes, a minority could be of little use, but the large majority would likely be driven to participate more, one way or another, even with basic income. After all, the basic income would merely serve that you do not die of hunger/thirst and can also enjoy basic necessities - which is supposed to be the common right of every human anyway. Perhaps then even an alcoholic bum could get a little roof above his head. To access more, work would remain mandatory, but it would no longer entail such a fierce, heartless, cruel competition as in capitalism.

There is danger that people ingest trends without much thought, consider progress taboo, like something that is necessary and how it should be the fastest, most intense that we can squeeze out from ourselves. Why? Why not consider living more moderately and in a relaxed tempo instead of chasing progress?

It is not about using the latest tech, but how society forces others into this, and, if someone else has a different opinion about what is accepted in the capitalist market economy - a stance that bows down to the trends -, then that person can expect that others will try to shame him, and outclass his opinion to the edges of periphery, instead of logical arguments that discuss pros and cons.

[OSNews](http://www.osnews.com/comments/30005) had this item recently:

> [The world has become like an eerily banal dystopian novel.](https://www.nytimes.com/2017/09/13/opinion/chelsea-manning-big-data-dystopia.html) Things look the same on the surface, but they are not. With no apparent boundaries on how algorithms can use and abuse the data that's being collected about us, the potential for it to control our lives is ever-growing.
> 
> Our drivers' licenses, our keys, our debit and credit cards are all important parts of our lives. Even our social media accounts could soon become crucial components of being fully functional members of society. Now that we live in this world, we must figure out how to maintain our connection with society without surrendering to automated processes that we can neither see nor control.

This danger has been written about before, extrapolated into a possibility of WW3:

> World War III will happen in the homes of individuals, who for the past dozen years have carried GPS-enabled phones around in their pocket while talking indiscriminately to their friends on open channels, as the owners of those channels recorded and analyzed everything. Since people willingly tagged their friends on the photos they posted on Facebook, thereby feeding facial-recognition engines, they even know what you look like.
>
> We will realize in hindsight that we engineered our own destruction because we were totally indiscriminate about who had access to these data and where these data were allowed to be. The technologies needed for data mining are free and open.
>
> It never seemed to occur to anyone that there could be a risk in this, or that war could happen within the homeland even as conventional armies were either here or somewhere else.
>
> In World War III and beyond, they won't use armies. They will use diabolical psychopathy, and they will strike at their target directly in their own home, like a thief in the night, armed with perfect knowledge of their chosen victim.
> 
> How can we turn this into World War III? It's very easy: just think like a bad guy. One fateful night, thousands of kids are killed in their bedrooms ... by someone who, thanks to their GPS-enabled smart phones, knew where in the house the child's bedroom was. This happens all across the country, and someone plots all of the locations on a map ... revealing the unmistakable shape of a scimitar. The entire country is instantly terrorized, troops are called out, and yet, in a few days, it happens again. Bewildered and shell-shocked, people will be then, when they realize that the perpetrators "cannot be caught", as their 'perfect organization' far exceeds the response capability of regular police, SWAT and other lawful forces.
>
> The nightmare of your worst horrors that have yet to come will make you _fear_ living all the time, since you will never know, "will I be their next victim at night?" You will not be able to protect yourself in the slightest at your most vulnerable state.
>
> We should have taken a sober lesson from 9/11, where three tall buildings in New York City, the third one being the center of the city's anti-terrorism response plans, were brought down with very exotic explosives that had been planted throughout the buildings right under everybody's nose. This is what "war" is in the 21st Century, not "industrial war" fought by armies, navies, and $50-million-apiece airplanes. In fact, all that exotic, expensive, and utterly conventional military thinking does you no damned good at all. Doesn't protect you in the slightest from someone who knows everything.
> 
> And there are two ways that we can find that out: the easy way, or, the hard way. Why am I certain that something like this will occur? Because of human nature. If you create a vulnerability, especially such a horrific vulnerability as this, then someone will exploit it to do an incomprehensibly horrible thing: truly, an act of War.
>
> Every major war develops after some technological innovation which makes war possible in a new and different way.

...That is a grim outlook, somewhat possible.

Now, back on topic...

Obviously, a paradigm shift is needed. A new social structure has to be formed, where distribution is much more fairer and just, and where the quality of life has the higher priority above the intensity of progress/development.

Social credit and mutual credit: 
[social-and-mutual-credits.zip](https://github.com/scientific-defense-force/this-is-the-world-site/files/1305695/social-and-mutual-credits.zip)

With social credit and perchance, mutual credit, and with demurrage, if properly planned and executed, then I reckon that game time (or full time) could be reduced for many. Competition can remain, but it has to be stirred into a different level or modeset. My point is, that people should not butcher each other over a loaf of bread or a glass of water; the basic, vital resources should be distributed in an impartial fashion to each and every one. They should also not have to 'sell' themselves in an overly-competitive labour market: since if some choose, for instance, artistic pursuits, then they should be free to do so without having to drag them down with a mandatory labour market, where they may or may not succeed - and if they would not want to or end up failing, they should have more options to pursue, is what I'm saying.

...

Societies today are so messed up, that people don't seem to want to understand other people, or put themselves in their shoes. Most notably, if someone does something they think is morally wrong (according to their own viewpoint), they don't try to understand that person's reasoning, they just flat out ignore him or stop associating with him. If more people chose understanding and sympathy over harsh judgement, the world would be a better place.

Even if someone is doing something I consider to be morally wrong, I can't help but being curious and want to understand why this person acts like this, does that or did that. Perhaps it's because of curiousity. Well, some have this craving to figure things out, to learn what happened, why, how and so on. Now, the way I see it, most people don't want to understand, don't want to improve. They don't want to co-exist and live in a better world, they just don't seem to care.

And if it remains unchanged, this will be our inevitable doom. Well, [Itachi](https://www.quora.com/What-are-the-best-Itachi-Uchiha-quotes) said that we only know who we truly are in the face of death:

> We do not know what kind of people we truly are until the moment before our deaths. As death comes to embrace you, you will realize what you are. That's what death is, don't you think?

I agree with him, since I also came to a similar conclusion. No more distorting illusions there.

Why is it that it's human nature to murder, kill and all of that (especially in the name of "God"), yet whenever you have a male-female couple with a significant age gap, it's sick and detestable? Though this varies by culture, but this ... Thousands of years of social inertia is proof enough what we can do. It is not hopeless to change this, it simply takes generations of time (and willingness to do so).

Unfortunately, you can't alter human nature at its core: what humans really are, how they function in general. What we can do, personally, is to try to have an impact on the few people or nature around us everyday. Probably that's all until you are not in any elevated social position: to become the change you aspire in your local environment.

What you would truly need, is a shake-up in human consciousness, mass consciousness, a drastic shift in their values. No big improvement is going to happen until then, no matter what scenarios you want to put into reality.

Life is not the problem, how humans express it, is. How people can hurt children and each other so deeply and leave scars that may never heal. How racist people refuse to learn to accept other races, cultures and beliefs, and stick zealously to their beliefs, even in the face of mass human suffering or extinction. That people have become so apathetic to the pain and torture of other human beings. We can't even focus on the real issues that threaten humanity.

The major problem here is that scientific and technological advancement does not necessarily imply human moral improvement. Now, an immoral or morally underdeveloped society with overdeveloped tech inevitably leads to self-destruction, don't you think? If you want to give a shot to a better livelihood, you first have to improve people's morality overall ... worldwide. For me, I can only attempt this at a local level.

Perhaps with a book, I can reach out to more people in my country, but that's about it. This is the responsibility for each individual. Can you see the immense difficulty here? Until people manage to rise up morally in the upcoming generations, you can forget about any worthwhile progress.

**Comment by AlistairB** [comment]
Thanks :)

> Life is not the problem, how humans express it, is. How people can hurt children and each other so deeply and leave scars that may never heal. How racist people refuse to learn to accept other races, cultures and beliefs, and stick zealously to their beliefs, even in the face of mass human suffering or extinction. That people have become so apathetic to the pain and torture of other human beings. We can't even focus on the real issues that threaten humanity.

> The major problem here is that scientific and technological advancement does not necessarily imply human moral improvement. Now, an immoral or morally underdeveloped society with overdeveloped tech inevitably leads to self-destruction, don't you think? If you want to give a shot to a better livelihood, you first have to improve people's morality overall ... worldwide. For me, I can only attempt this at a local level.

My personal take is that humans are not evil or good, but very er humany..

Humans are exceptionally flexible in terms of absorbing culture as they grow and moulding their behaviour but beyond that we generally act in similar ways and have a similar goals. https://opensocialism.com/open-socialism/arguments/what-motivates-people is my attempt at defining this.

I don't think these drives have changed much over time and I don't think changing humans is required to improve our society. More so we need systems designed with existing human morality and limitations in mind. We understand ourselves very well and have amazing technology so this should be feasible.

> Yes, a minority could be of little use, but the large majority would likely be driven to participate more, one way or another, even with basic income.

Agreed. I think the idea that people won't strive when their basic needs are met, or won't strive if there isn't the potential carrot of becoming super rich is not valid. Humans are biologically programmed to promote themselves in some fashion and attempt to attract a mate. As long as some means are available to do this most humans will pursue it regardless of any other factors.

**Comment by niltwill** [comment]
Alright, some more thoughts.

What if I want to disengage from your system, shouldn't I be free to do so and tend to my own life without your interruption in my personal area? I would prefer if people would be free to leave and enter conventional society, if that is what they wish. No system should be enforced for participation.

...But I still choose too exist in your system, because it's easier for me there. Now, say, I have a bad mood for weeks or a horrible life to lead and can't be bothered about _your_ nonsense. Sort it out yourselves, because I'm just this selfish. How-you-like-that? I'm your average human, thus I leave these dirty games/politics in the experts' hands to deal with. That's why they are elected. If I'm also forced to vote or participate in these matters instead of those who represent me, then shouldn't I also have enough expertise in this field to know what works and what doesn't, basically, to have an understanding about what would be the best option to take? But I don't. My vote then can be easily manipulated because I'm not an expert, and I'm being led by the nose.

Let's assume I'm an average farmer who has no understanding of the region's needs. I produce my own crops and sell them at the local market, I don't care about other things or the world or what-have-you-in-mind. I vote for the guy that has the expertise for it to represent my region, even if he is a sham cheater because he manages to win me over with his speech. Now if many more do it like me, then the consensus can be turned against the interests of my whole region or country or people.

Common folks like me do this without ever really thinking of why I am doing it. Perhaps there is now a monthly referendum (in the community centre or something) where I vote for the most sympathetic guy. Will it be my fault then if the representatives ruin our country, though I don't want that to happen? Isn't it because of their deceitful manipulations?

Even if we, the regular people, can manage to impeach the sinners and place new ones in their place, then what if in every turn we end up with the same corrupted bastard for decades? Because we inherently lack moral and charismatic leaders? That those who could truly lead us to prosperity and act in our benefit are in the very small minority, rather than the majority in the representation of us all? You would have to go through many iterations to weed out the waste before encountering a diamond in the rough. A good, trustworthy leader is very hard to come by, is what I'm saying.

Without better education and a moral evolution (oh, and a system that does not reward corruption), chances are those who could work for our benefit will all be gone before they could make a single dent, in the upcoming future. So we'll be left with those madmen that we had in history all along. Those who will lead us into death and suffering. Know that the world was more compassionate years ago, so the slow decay is apparent. Not all hope is lost, there is always much value in living, regardless of how rotten society becomes.

> we need systems designed with existing human morality and limitations in mind

Sure, then let us operate with the current stage of human morality and limitations and see how far we can go.

Now if you have some experience...then you know that people (right now, in this era) can't be trusted to do what's in their own best interest, never mind the interest of others. Therefore, you don't give people money, you give them services which satisfy their basic needs.

Remember, we are dealing with average people here, the ones who make up the bulk of the population, who have - unfortunately - fallen victims to the indoctrination of the media/gov and whatnot. The government basically brainwashes them with learned helplessness. Would you be striving to work if it doesn't make any difference and when you can barely sustain yourself on some mercy morsels? Sit on benefits, or go get a job and lose them, be just as bad off or worse, and have an aching back or some other physical/mental issue(s) besides? (Don't you think this gives them a great incentive to take on a living as a criminal instead?)

Honest people in this system can work themselves to death and no one respects them in this wicked system. "Trample on others or be trampled" is the modus operandi. The lesson is clear: society doesn't want you (those who live on benefits) to work; we'll pay to just keep you quiet and make sure you stay out of the way. It's better than prison, which is where you'll end up if you discard of any of the other 'given' opportunities.

Most of these people with a UBI (Universal Basic Income) would knock themselves out with drugs, alcohol...with that flow of basic income. So I do not support the idea of giving them money. All what you'd get is that people would consume more of the latest fashion or trend, such as the lastest iPhone or whatever. Do not expect them that they will all use that money for their everyday necessities.

So for a set amount worth of money, you could, instead, provide everybody with clean water, some basic nutritional food, free laundry, a little dwelling, and perhaps a free visit or check-up regarding healthcare (though it may not be optimal or sustainable for all services to be free)...**give them services and products, not money.**

So you could call it ... the Universal Aid Package (UAP) instead, and what should it contain? Agalmic goods. Stuff from overproduction is a good start.

Money is only a means of exchange and is not a necessity. The main point to consider is that you can't just give them cash. Many didn't get into that bum-like or poor or indebted state because they're good at handling money.

If you want to offer help universally to everyone, you have to make this fool-proof. So you have to offer services or items that EVERYONE gets (regardless if rich or poor, different culture or skin color - the point is not to discriminate under any circumstances), rather than money.

This could come in a new form of "service points" or something. It could be similar to an issued card. You have to spend it all or it will get taxed and begin to lose value/points after inactivity of a month or two (though this time should be tweaked to fit the needs). You get a certain amount of points per month as per UBI, and services may cost whatever amount of points as well. If you reach zero, then you would have to begin to pay for it. You could only use those service points for very specific services and products (only the most vital resources and perhaps some extra services such as healthcare that might cost more). It could be a start. But then again, I only made that up recently.

Besides the service points you could use up in shops and elsewhere in society, you might also get a basic Universal Aid Package with some rations, say, per every week or so.

> I think the idea that people won't strive when their basic needs are met, or won't strive if there isn't the potential carrot of becoming super rich is not valid.

Indeed. Consider the following: do you think people would think earning $200 (or some fixed amount by which you can barely get by) a week/month is better than earning, say, $1000 or more? There will always be people that don't want to work, it's just whether one system means the people that do are catered for better. There are those unfortunate people who want to work, but find themselves unemployed through no fault of their own.

I reckon that most people would keep working if they own an UBI (Universal Basic Income). Just check out how many women and men are volunteers, or keep active when retired.

However, I explained before why I don't think giving them money is a good idea. You want to offer them services and some items as a basic living right, access to basic resources, not to seethe further conflicts with money that can be spent arbitrarily anywhere.

**Comment by niltwill** [comment]
All this being said...I think this is a conundrum with no solution.

I dabble in IT. So I have a dark view on how the future could look like from that aspect. It is the very base of ultimate collapse.

Reality is always getting further away from ideals. Do you remember when it was all about "how the Internet will establish a wonderful world?" That knowledge will reach everywhere and so we will become the wisest and the most prosperous society? Naivety.

Now, do a reality check...

The kings of the internet are not the videos about sayings of wise people, but those with cutesy animals (cats); it does not deliver the western values everywhere, but it aids in training terrorists and spreading hatred and conflicts, and it is not the deliverer of free speech and the smooth ideas, but it's increasingly more about pushing certain (political) ideologies and other opinions will be censored or deleted. This is what [net neutrality](https://www.savetheinternet.com/net-neutrality-what-you-need-know-now) stands against.

["GoDaddy, Google blacklist Nazi website Daily Stormer"](https://arstechnica.com/tech-policy/2017/08/godaddy-blacklists-white-supremacist-site-dailystormer/)

Niice. Let's silence those, who do not think like us because they want to silence those who do not think like them, to show them that silencing those who do not think like them is wrong...very efficient. It does not matter how vile the nazishit is; a contract cannot legally be voided based on political views. Discrimination is illegal when done on basis of ethnicity, sexuality, religious convictions and political convictions. I'd rather the net would not go down on this slippery slope. Because if it does, then sites or videos, etc. that provide alternative views from the rest of the main media or preferred ideology will end up getting banned, right?

Now how nice is it if you - just for fun or to annoy someone - teach your dog "Sieg heil!", upload this in a video on YouTube and then you might risk jail for 1 year? This is what this denying game leads us towards to: now, you will have to be very careful not to do things the current regime does not approve of. To always be mindful of your actions and words...Which is extremely taxing and overbearing.

> It would be extremely bad for registrars to become actively involved enforcing domains are only used for "politically acceptable" views. While it happens to be against some despicable people this time, it's a very slippery slope. If we allow registrars to take down domains because of what the owner says, then we will be weakening freedom of speech for everyone online. We all need to stand up against the hatred, but it's a really bad idea to have registrars deciding who's allowed to register a domain.

This touches the topic of censorship.

> It's a short and slippery slope from benevolent intentions to fascism. People do not need to be sheltered from information. If they lack the ability to exercise judgment, then they should also not be entrusted with things that require responsibility, like cars, guns or children.

Big tech companies have made up their own ideologies (or took them from society's cesspit). This will eventually grasp into your life as well. They desire more to enslave you and make you dependent on them, and if you do not follow or share the given ideology as expected, then goes the punishment. If you advocate [communist ideals](https://fee.org/articles/liberal-socialism-is-another-false-utopia/) (or wish to use this as a starting point), then you'll have it soon enough and will experience first-hand how 'wonderful' the delusion is. They no longer want to sell you a product once, but wish to continously supply you forever; so what once you used to pay for once, you can now keep on paying again and again at certain intervals. Or else they will switch off the service.

Or "why not let us store everything in the cloud for you, that will be so much better for you. Don't you keep your essential things next to you, or in your residence, but give those to us and keep paying to us forever." They should have household appliances on the net and make sure it's designed in a way that it will not function without the internet, but you can enjoy listening, unavoidably, to advertisements, and you will not be able to avoid how the product is going to collect legal data about you. You'll have no choices. Either obey everything that we ask from you...or we'll make all of your things unusuable.

The horrible future is such that the dream of tech companies will come true. So it does not matter if we are aware that the hacked 'smartcar' kills the family, we'll buy and use it anyway. We might know that it is not a good idea to control a nuclear power station through the internet, it will happen. It does not matter that we know how much harm we cause to ourselves, we'll do it - so there will be Industry 4.0 with a factory that is connected to the net, even if hackers bring companies to ruin. The state also handles everything through the net, despite compromising the very important, confidential data.

Simply everyone will betray everyone just in order to make the dream of tech companies OR mindless pursuit of the market come true, they won't even care if they (themselves) die. Of course, the tech companies will bloom. It does not count how there will always be some sort of a grave catastrophe, since the people, much like zombies, will keep desiring their products, and will no longer say and decide that "hey, now I will buy the traditional car that cannot be hacked; so many have died before me, why would I take this risk?!" No, people will no longer consider this.

People, after their child has died, will buy a new smartcar and everything will go on in this crazy fashion. They will simply procreate a new child as if nothing had happened. Add on to this identification chips planted in at birth and robotification of humans (I bet they will find a way to turn off the emotional system and alter the nerves so they won't feel pain either), and there will be a nice borg fief for the Kingdom of Rulers (aka. Neo-Marxism, coming straight outta America first). Until it all gets wrecked by nature and the "collapse of industrial civilization" as we know it. I think this collapse is actually spot-on. I only hope it comes sooner rather than later before all this madness unfolds.

The future cannot be stopped. The nuclear power station will be on the net and if it blows...well, bad luck, but the radiation will eventually dissipate, right? If we have to weigh between comfort and security, then comfort wins, since it is only a mere nuclear power station, bringing that down cannot stop the future. The airplane will become as such that if it crashes, then a new one can be produced, which increases GDP and the new will then have the security patch for the issue that made it crash. Then another one will hit the ground eventually, getting a similar security patch, to calm down everyone..at least temporarily. And this is how it will go, because it will be unimaginable that their control should not be connected to the net. Same story with technical or industrial things, the companies will bear an endless amount of expense, rather than saying: "I don't connect this to the net, it's not worth it."

One can only hope that sanity occurs before the nuclear power station also reaches the 'New Industrial Revolution'.

Would you buy a car, where hackers can regularly gain their control over? Would it be calming that you would have to regularly update its system? But sometimes a ransomware or a virus can chime in and this would mean you will no longer be in control of your car and perhaps you either pay 10,000 Euro/Dollar/etc. or you'll be driven with high-speed to a tree. Of course, after your death a security patch comes to this issue, just as how the manufacturer released similar patches during the prevous months due to similar incidents. But there is always a newer and newer one, and they are alwasy one step behind the bad guys.

Or perhaps in your smarthome, certain kitchen stuff want to do the shopping instead of you...and of course this won't be defendable and your money can be spent by someone else. Wanna risk it? Not in the era when almost no one has these items and you barely have anything to fear, but when it becomes a regular and common sight for almost everyone to have them...

To put smart spy devices in your house, which will yell advertisements even when you want silence or wish to sleep? To make you pay for silence every day? In the future, when it's the billionth case being dealt with that the cloud is not safe at all, then why would you trust it with your important data? When political parties are more and more extreme, would you like your government to be aware of your every little secret? Would you like to constantly keep on paying hefty sums for things that you only have to for once nowadays?

...

These times, it is becoming less and less important for a human to be talented. Companies couldn't do this before or they would have gone out of business. But now in this new age, which is famous for its lack of creativity, one can sway into ideologies. More and more tech company is becoming an utility company, so they no longer require creative people. Since whatever exists, that has to be maintained and that will generate money again and again. Storing data in the cloud, requiring a monthly fee for that, spying on what the dude is doing on his computer and selling it to ad companies, and we shove ads down his throat, etc. this does not need talented people. A cloud company does not require creative people, they only need maintainers now.

...

If we entrust everything to IT, then there is going to be a sword hanging above our head (constantly), because we will not be able to abolish its vulnerability, and the attackers will always be ahead of us. And if the attacker's goal is to crash a country, then he will be able to do it.

We cannot prevent them from gaining access into the system, not even with the latest über tech. Therefore they should never connect important systems to the net. Besides, if you put everything to the cloud, then when the energy source depletes, how will we be able to access all those data?

It does not matter what kind of security measures they have in place, experience has shown that every system is hackable, it is all just a matter of invested resources. If a country wants to ruin another, then these resources are at their disposal in most cases. And compared to traditional warfare, it is much cheaper to cause harm with this. Then **why do they plan to handle every single state matter on the net?** It is such fun for the sake of comfort, to have a sword above our head, which we are constantly aware of - and that if they want to, they can strike us down?

If our state will be wired to the net, including the important elements of the economy, then it is guaranteed that our lives will remain nice for as long as our enemies also think so that it benefits them. They can refer to "national security" in order to steal our inventions. If things get rougher, then it is not about spying anymore, but crashing.

The tech companies will shove it down on people's throat, and exultantly exclaim their deed, that they've done it. They've waded common sense, went beyond reason, and ignored all warning signs. Thus we have entered a blind alley and tech companies will learn the hard way (their triumph will be their downfall). This negative utopia only lasts for as long as another country does not decide to wage war with us, in fact, since it is very hard to discover who attacked us, they can attack us covertly.

They may be fine with just stealing anything they can, from us. The cloud would exacerbate this. It is enough to gain leaks from one cloud service to gain access for millions of secret data. Isn't that nice? They don't have to do it one by one, and they gain data about extra people as well.

All security measures have failed, whether in small (virus cleaner) or large (Pentagon). And they will stumble in the future as well. In the future's case, however, the ads or ransomware will no longer be displayed on your computer, but your current supply will be turned off and your car will be driven into another. Since companies are not normally regulated, they only chase after dirty money. How cool if everything will be in the cloud and you will have to pay a monthly fee for it. "Wow! We no longer have to care about real developments. The user has no choice but to pay, and we will sustain ourselves forever..." That's how they may think or what they believe. There is also the motivation that the U.S. can spy more easily, so it's not only about money.

In this future, the tech companies would profit in short-term, but their end would be guaranteed. After the downfall, people will turn away from everything. Especially from tech people, who crashed the whole civilization and set it back into the ancient times. Everyone loses. Even those who made the Western Civilization fall will lose, as they stopped the progress of humanity.

The digital future will not have a long lifetime. The question is whether it will crash everything down with itself, or only itself will not be able to bloom.

State secrets should be properly shielded, so they should be separated from the net. And about the economy...they say that our resource is knowledge. Then why connect it to the net, do we want it to be stolen? The IT is vulnerable, and if we connect the whole country to it, then the country will also be extremely vulnerable. They have to reconsider what can be connected to the net and what should never be.

The future is pure madness, any way I analyze it, I see only downfall.

There is a certain trend that they wish to eradicate all traces of paper money, and handle everything electronically. Basically, to make money into an electrical signal. Don't they realize that IT is too vulnerable for these transactions? Even the giant banks can be exploited successfully. If countries get into conflict, do you think they will march into physical war with each other? Nah, they can just hack the other country's inner systems. The end result is that your money, everyone's money, and perhaps identity, and basic info...might all be gone. And then what will happen to us?

Two-factor authentication does not help too much either. Smartphone is a huge vulnerable beacon. The software that the phone uses can be exploited, the touch-free tech can also be read from far away, and the store's system (such as Google Play) can also be a source of danger, that can also be abused. The whole smartphone is a big joke when it comes to security and privacy. It is a mistake to trust your dinero on this tool. I don't like the tendency to open up your whole life in front of companies; yet the smartphone is extremely successful in achieving that.

People do not really realize what it means to lose their private sphere. I dread the age when you can be assaulted on the streets and through hacking your smartphone they can drain all your assets and everything you possess, whether you have the money in the bank or not. And touch-free technologies to a certain amount do not request any security confirmation. If they steal it, then they can steal all that money up until the limit.

And is this really more convenient and is it faster? If someone uses a bank card to pay, then have fun waiting. Even the system can shut down or have anomalies. What would happen if only electric payment would be allowed, and the system shuts down for only a day? Huge tolls would rise.

Make no mistake, our rights (of privacy) are being violated every day, not only on a local but on a national level.

If you made a movie in which enemy agents were watching a million people in real time, they would call it a thriller and they would assume that it was fiction. There will be hell to pay when they comprehend that it is not.

If someone put up a web-site with a webcam photo of Times Square, such that you could point the cursor at any person on the street and within a few seconds (if that person is carrying a smart-phone in their pocket) start dumping anything of what is actually known about that person (known by someone, somewhere, anywhere), there would be an unholy explosion. It has not yet dawned on people that such an application could, right now, actually be built, and anyone in the world could use it. So who's gonna break the news? And, what do we plan to do when someone does?

We are living in interesting times where virtually anything is possible, but where we are not starting to ask, nor to attempt to answer, the hard regulatory questions of what is, and what is not, okay. When you send a letter, the post office doesn't steam-open every single one. When you make a telephone call, there are (or at least supposed to be...) laws against wiretapping. But every form of electronic communication that you send over the Internet is supposedly fair game. Likewise, scanners read your license-plates and apparently have no difficulty at all connecting that plate to you. They tell you "it's for your own benefit."

Notice that this is not the state's doing. These are profit-seeking private corporations. Or, at least, this is what they claim to be. And there are no restrictions on what data may be gathered, from what source(s), or what may be done with it. This data isn't top secret. It isn't secret at all.

What if some company snoops out your habits of what items you usually buy with a credit card - AND transfers your e-mail address or data to a dog food company (if you buy tons of dog food), which begins to send you spam e-mails, and others..Now, a credit card has become a hard requirement in the modern western world.

End result? You are dependent on the bank's mercy of its service.  However, if the economy is stooping low in a country, you may not be able to transfer your money from your bank account/credit card even though you earned it - there may be a limit of how much you can draw out. So you become limited in what money you can access. And that is only one thing. Do you think banks cannot go bankrupt? Were that to happen, your money might also get lost in the null void - poof, gone in a moment. Same thing if a global or world war would occur (or if there is that upcoming economy crash or the collapse of industrial civilization).

You can argue "but hard cash is troublesome to carry on with you"..."you can easily get robbed" and "using a card is much more convenient than cash money"...Yet is it any better if ALL your transactions can be traced back? Best is to use an ATM and transfer the money on your bank account to cash money. But there may come a law someday that bans ATMs and mandates the sole use of bank accounts/credit cards...so there. People don't realize how limited their options have become until there will only be one choice remaining: passive death and suffering, any active resistance will be impossible!

A lot of people think about privacy but don't really care until something happens to them personally. **It's like freedom. You don't appreciate it until it's gone.** If you are a victim of identity/money/existential theft, you experience a change of world view, you realize how little control you have over your world.

And now, camera-equipped model airplanes are flying overhead, and companies want them to be protected like commercial aircraft. A company is excited to be able to emit sub-audible sounds from television programs which an app in your pocket, unknown to you, can detect in order to inform an advertiser, without your knowledge or consent, what you're watching. It's easy to see that the company could simply record, and analyze, what you are saying to one another in the comfort of your own home.

When someone uses the Internet as an instrument of terrorism...And they will, to commit atrocities never before imagined (even though they are, in fact, quite easily imagined), then there will be unholy-hell for us to pay. Information is now available everywhere, to who-knows-who, and do we really have to wait until after something big happens to know that something's wrong?

People for hundreds of years have been successfully marketing things to people without knowing every theoretically-obtainable detail about them. Wouldn't it be nice if the only thing that happened is a price-change on a pair of pants? What if, instead, a data miner from 10,000 miles away determines exactly what your habits are...Exactly when your aunt, say, is home alone and they know that the surrounding houses are also empty, and they walk up with a hunting/sniper rifle and then walk away, knowing that there will be no witnesses?

What if this started happening all over the country, at totally-unpredictable intervals, revealing that the perpetrators weren't talking to one another but that they somehow possessed near-perfect information about thousands of people? Wouldn't that mean, like, war?

If your purpose was to lay an enemy's country low, you would read The Art of War and take it to heart. You would know to "strike where the enemy is not looking, and in a manner that he does not expect." You would also know that you have unlimited time to prepare, not because he couldn't see what you are up to, but because he refuses to do so.

"I did not do anything, because they did not come for me. Then, they came for me."

If we allow them to observe us, and if other secret service also monitor us, then they can steal our inventions, business secrets, or can even crash the net-obsessed country in the name of "national security."

In our age, the human knowledge is formed throughout peculiar channels, that filter certain thoughts, manipulating people. The TV, computer, tablet, smartphone, all these constantly have an effect on people. Many say that if people can freely access to information about the world, and can share it between themselves, then a wonderful world will arise.

But social conflicts are coeval with human civilization. The source of conflict is not just information, or the lack of information sharing. Not only those societies had conflict who had no knowledge about each other, but also those who have well known each other. This is true for individuals as well. People can conclude things even if the same information is available to them. Where the different values, ideologies, goals confront each other, then showing these and trying to combine them leads to confrontation rather than to appease conflicts.

The Internet brings the discordant ideas to closer relation, it did not appease conflicts in practice, instead the extremities gained more publicity. On an international level, the - otherwise attention-lacking - small extremist groups gain worldwide attention.

Connecting the communication technology with the Internet has a very inherent danger: the individual will become less and less capable seeking for memories and data in his head, since his thinking is also becoming more and more reliant on technology. The Internet reduces our historical recollection. The reason is the following: the human forgets all that which he believes he can access later, and remembers all that which he believes will no longer remain accessible later.

The Internet reduces the urge to remember, by always giving us the opportunity to review things later. Since information is always accessible, the human psyche feels less need to memorize things. This is why humanity is slowly becoming defenseless against the information centers' regulators, and by removing this opportunity, it can endanger civilization. Easy access to data can also be a positive thing, and helps the work of researchers and scientists, but that's another story.

What's the problem in the way we gain knowledge nowadays? Well, some of us like to think deeply, and thus feel the need to make independent decisions and for independent thinking. So we refute popular thoughts, such as the "cloud is the future", which is merely the manipulation of some big companies. Some of us do not merely require facts to acquire, but a knowledge of a bigger picture. The Internet did not elevate us to the society of knowledge yet, but rather altered the thinking of most people; so that answers merely have to be searched, and the solution to a problem no longer requires consideration and reasoning, but just searching (after all, we believe the Internet should know everything).

So the Internet does not make us wiser, but it makes us concentrate on the facts, on the information itself. But that by itself does not make you capable of processing such prepackaged information. How much of it is true? How many ask this question? How many doubt what they read? This is how people can be baited and manipulated. One should be able to place these facts in the wider context of experience, to make them really usable and thus to create knowledge.

The telecommunication impaired another form of traditional communication, which is personal communication. The chat eradicates a large chunk of the emotional-psychological dimensions (which have been there for millennia), which play a vital role in establishing healthy personalities. Nowadays, the Facebook culture strangely results in us becoming less capable and dedicated towards personal relationships.

People thus escape from emotionally challenging situations and thus break up in an SMS, for example. They also rather rant on the web with strangers on forums, rather than talk about their problems with personal relatives.

More and more personal conversation lands in the hand of some big companies. (Think of mobilephone-obsessed people.) And these conversations remain forever in the age of the Internet, and who knows when they can use this against those people. It is also easier to manipulate crowds through this.

> Data companies seem to have decided they should meddle in, being the arbiters of truth and morality as a replacement for the critical thinking and reasoning. Embrace the future where the corporation does all your thinking for you.

So who decides what is agreeable and what is disagreeable? What is fake and what is truth? There can be technically fake, but morally enlightening pieces. The censorship would make sure you can't joke around or deliver sarcastic or satiric or other spicy messages. How can an algorithm take into account what is fact and what is opinion? GG programmers.

> We're in a situation where you have data giants controlling the data - and shaping it the way they see fit. Ordinary people are not privy to the monetary, altruistic or other considerations there might be here, they only see the end product. This means that the landscape of information that people consume is directly impacted by algorithms designed to make data companies bigger, stronger and more profitable. Data manipulation is already happening on a mass scale.

> So what we end up with is a theocratic model - a Web 2.0 religion but a religion nonetheless - where companies dictate what idiots should see when they browse about. It's not just the matter of how accessible the information is, it's also about deciding whether the information is good or bad for the masses. That's pure propaganda. Besides, what credentials do Internet nerd companies have to peddle their morality model onto the wider world population? Apart from the obvious fact that whatever the data giants do will be very US-centric and thus irrelevant to some 80% of the globe, I don't need anyone imposing their moral compass and beliefs on how I should access and use data.

> Moreover, this model removes direct responsibility from people. If someone else filters information for you, then you don't feel like you need to do it yourself, right? Plus the model is horrible. And useless. We already have so-called "naughty" filters to protect "children" - it's always "think of the children" model to tickle the guilt glands among the billions of clueless cretins - but never mind the fact you can see ten executions of people or brutal murders on Youtube, Liveleak or 9gag at any given moment. In the worst case, you need to "sign in" to see the brutality so that companies can profile you and then send you advertisement and recommended videos featuring more of the same horrible nonsense. Woe the random boob, you want the violent stuff!

>  If companies want to "protect" people, then they should also be criminally accountable for when they fail to do so, the same way governments are. You want to filter out crap? Good. But then you will also be held accountable when someone sees something they shouldn't or don't want, and if someone downloads a movie because there was a button, then whoever put the button should be asked questions. Or fined. Or jailed.

> But it can't work both ways. Either you give people EVERYTHING, and then they are in charge, or you control what they get, but then you are in charge. Zero accountability model does not work. Except as some sort of stupid morality banner under the guise of political correctness and similar dross. 

> When PRIVATE companies start playing this game, it becomes dangerous. They have no regulatory rule in people's lives, no obligations, no responsibilities, no liability. No one chose Mozilla or Google or Facebook or Baidu or any other company to be their moral compass of the day. Hi-tech companies can play their kindergarten values in their cubicles, but not out there in the real world.

> **The real danger in this model is - if this "voluntary" censorship sticks, then next step will be profit and then "benevolent" manipulation as supposedly negative data streams that cause people unnecessary "mental burden" are removed. You end up with brainwashing.**

> Truth initiatives are a horrible idea, because they imply there is just one truth and that it should somehow be accepted blindly. What this is going to do is make smart people even more suspicious of the nonsense the media tosses at them, and the stupid people even stupider. Maybe that's the goal?

> I do not need pseudo-liberals to be my tranquility police. I do not subscribe to arbitrary values of goodness, because such a thing does not exist. The walled-garden mentality is a horrible thing, and it's been done before, throughout history, in countries, societies and regimes that do not resonate well with the so-called democratic process.

> The greatest human trait is curiosity. The need to learn and challenge conventions. We got where we are by fighting the established truths, by coping with the unknown and uncertain, by not accepting the reality at face value. This is just the digital version of going to the middle ages and following the party dogma or some similar nonsense. One truth to bring them all and in the stupidity bind them. In the land of idiots where the data lies.

[[source]](https://www.dedoimedo.com/life/censorship-1984-dot-now.html)

...

Lewis said:

> Of all tyrannies, a tyranny sincerely exercised for the good of its victims may be the most oppressive. It would be better to live under robber barons than under omnipotent moral busybodies. The robber baron's cruelty may sometimes sleep, his cupidity may at some point be satiated; but those who torment us for our own good will torment us without end for they do so with the approval of their own conscience.

Then Tolkien clarified:

> We may indeed in counsel point to the higher road, but we cannot compel any free creature to walk upon it. That leadeth to tyranny, which disfigureth good and maketh it seem hateful.

Like Tolkien, I have a strong moral opposition to compulsion, even the compulsion towards "truth."

There may be a path that is unambiguously "right", but free creatures should have the freedom to choose whether to follow it, or not. To deny that freedom would conflate the "right" path with the wrong.

Tyranny maketh good seem hateful, because compelling a free creature to walk the "right" path would ultimately drive them farther away from it. The "good" path would seem evil, because the all-powerful 'overlord' is forcing you to follow it, and the "evil" path would seem good because you're choosing it of your own will.

Freedom to sin is an important part. Removing that freedom (forcing them to walk the correct path) removes the truth of their redemption (because they did not make the choice). It will demonize the "good path" because it is a thing you are enslaved to, and so one will eventually rebel against it.

Also the act of forcing another to the right path corrupts you, for you are enslaving them to it, and removing their free will. Thus you become no different from the likes of Sauron. Even worse when the tyranny is exercised for the 'greater good'.

> Gandalf as Ring-Lord would have been far worse than Sauron. He would have remained 'righteous', but self-righteous. He would have continued to rule and order things for 'good', and the benefit of his subjects according to his wisdom (which was and would have remained great).

That's about it.

...

In the future, they will look back on our era and will wonder, "how could people have been so stupid and narrow-minded to waste away at such golden opportunities?"

The West gained what no other empire could: obvious world mastery. But this thing melts away super fast due to many things. What is one of its greatest advantages, the technical development, melts down, since others are allowed to steal their secrets. In fact, it assists them. Thanks to the "power" of the cloud, nowadays even a whole continent's every company's secret compilation can be stolen away by, say, China in one action.

If we would seriously consider data security, then the cloud should not be supported, but banned. According to Zuckerberg, the digital age's oil is information. If so, then we constantly allow this to be pumped out from us. The lack of cybersecurity (protection against cyber threats, see recent ransomware crises) causes drastic harm, but people don't really mind.

The many lazy and simple people who do not want to rot in an office will lead us to deploy some sort of a digital state; business employees, owners, bosses, who only care about dirty money, will make sure that their project stands at all costs, regardless if it's crap from the aspect of security (and usually not just from the aspect of security). State governors who are corrupt and chase after popularity, will show how modern they are with this digitalization-obsession.

Make no mistake, **we live in an age when almost everyone is digging the grave of their people's/culture's long-term interest**. It is possible that the thunderbolt will hit in their own generation already, or perhaps in their children's lives and that will be the end of the West (and industrial civilization). That is the future. I know it full-well that there is no correction on this path. It is what it is.

They say that the West is like the Roman Empire at its decline. In my opinion, it's much more like the fall of the Spanish Empire. Which is not that well-known.

Today, Spain is a third-rate European country. It used to have potential, so that it could have been what the US was at the early 90s. A superpower. It still has its signs of its former superstatus. According to an assessment, more people spoke Spanish than English back then. It could have been a blast. So what gives?

Both the Spanish Empire and the wealth of the West was forged by hard work. Then it merely took a lifetime of a generation, for everything to go into the direction of slow decay and disintegration, instead of further improvements. People became comfortable in their well-being, and lazy, and started to barter away their inheritance which would have ensured that not only they would have it better, but also those who came after them.

At the Spanish Empire, basically, due to the big prosperity and the military "honor" (and perhaps, making sure how everyone feels and acts like a noble), this caused the death of its own industries and agriculture. And they became used to the comfort so much that they'd rather starve than change. They considered mundane work to be undeserving as the offspring of heroes (or nobles). The whole tragedy of the nation derives from this pride and presumption. They would rather starve to death than to do work that they think befits that of a slave.

Nowadays, almost everyone needs a diploma, even the street sweeper. Presently, there is a similar trend of how everyone is aiming to be an "intellectual." They have to be, to enjoy the cocktail! And yes, even today, they leave it in the hands of foreigners to do the "what I don't like to do" type of work and would rather pay for it extravagantly than do it themselves. They strive to give everyone a diploma...of any kind. People take up mortgage and credit, to look cooler.

The population dwindles, and the incoming people they want to supply the economy with are more of a burden. Much former knowledge is lost. Many industrial stuff has been taken to Asia, and those who had the expertise did not pass their knowledge in the West for a newer generation. In the Spanish Empire, they broke their habit to do things themselves, since they could purchase whatever they wanted in the "golden age." It is similar also how many people used to support many inactive ones. Also it is now noticeable, how many people grow old next to their parents or in their own flat with a cat or a dog, while wasting their lives away on social media (most notoriably, Facebook and Twitter).

The West gave away its security for comfort. Thanks to it, it raised into a superpower status China might wage war against it, and the cost of comfort might be the total downfall. It gave away its industrial knowledge, and all sorts of its business secrets, since this digital world is so comfortable, that who cares if they steal everything away from us, it's fine to us if the world's biggest thievery is ongoing everyday. A decade worth of development; upload it to the net, let them steal it. We put the knife to our neck, we build the vulnerable, digital state, and so we'll make, for instance, the current supply exploitable, because it is so comfortable. People lack self-defense skills, but we continously let in those seasoned or trained people who hate us, in fact, they fill up the German army, for example, so if there is a grave situation that requires armies, it might attack its "own country."

The West is also full of all sorts of foreigners, who do the 'bottom-level' jobs instead of them, while marriage is disappearing and the population number dwindles. As an extra, we will also have our lovely, costly, expensive, inner enemies as well. There is also the huge dependence on foreigners.

These processes have been revealed by analyzers back then IN THOSE DAYS, but they could not stop the downfall, the consummation merely required a generation to pass. Nowadays, the West has unprecedented education and lessons, many intellectuals and universities. They won't help. It does not matter if they show us how very bad it is whatever is ongoing, or what people ought to do - everyone is chasing after short-term gains, so the future will not be bright. This would not be necessary, we would simply need people who think and act with responsibility, but the days today are not such times. This is true for IT as well...

However, Bismark said: "I am firmly convinced that Spain is the strongest country of the world. Century after century trying to destroy herself and still no success."

The West is similarly working on this goal day by day. IT helps in that considerably. Many trades are lost in the West, and foreigners come to do the dirty work...

This world is such that you either learn from history's lessons and mend your ways or perish forever.

...

This is my last long writing here, since this topic does not warrant any further approach IMHO. My stance is clear on this one and I'm done with this.

Humanity ditching its primal nature is not going to happen. It's not wired in humanity's biological make-up. Instead, lustful urges, like sex and vengeance, always band people together and help them survive. 

Humans are still violent apes and borderline killers (with a little ethics for some of them to boost themselves up), not some intelligent species. I'm telling it as I see it. In general. A rational being would not go ballistic upon the slightest grievances or annoyances. Some humans can even kill you for it - or if not that, then exclude you from their society or social circle forever.

Or, for instance, consider our meat industry. How can any empathetic human, with a heart, snatch away an animal's child from his mother and eat him? How can they have the heart to feed a sentient animal with so much starter that his liver burts? Yeah, I know, "this is how life is", "the stronger feeds on the weaker".

We see this in the animal sphere, but aren't we humans because we CAN rise above these bestial or brutish things? Of course, for that to happen, first we would have to toss aside the bestial things among us humans. We can't even do that. We can't learn to respect and love each other, we can't treat others humanely.

Only AFTER we learned to treat others humanely, could we feel empathy to the weaker and the more naked (also called compassion). This is what being a human is all about, to restrain yourself from wantonly killing animals out of pleasure, or to restrain immodest passions.

Why is it that it's human nature to murder, kill and all of that (especially in the name of "God", yeah people do all sort of wicked acts under God's name), yet whenever you have a male-female couple with a significant age gap, it's sick and detestable?

The predators can't claim that they are moral, they do not fight for the rights of women, minorities and others, they do not have nature-preserving associations, they are not shocked at an animal's torture, they do not advertise the protection of the weak and oppressed, etc. What humans do, is hypocrisy. They paint themselves as intelligent, morally superior beings, but next to the table they are real mass murderers. Many people even kill out of enjoyment ... or pride.

In this conundrum, I don't see any solutions. It's a fun exercise, but ultimately I don't care either, if others (the rest of humanity) also don't. To me, this is to scratch an itch. I'm stuck here for whatever reasons, but I admit I don't particularly enjoy living or like what I (have to) see in the world (and the powerlessness to change anything).

Sometimes I entertain the thought that it'd be better if doomsday would come after some time before Industry 4.0 and engulf us all in death already. That'd be more of a blessing than letting all this immense suffering play out. I don't see any chance for redeeming humanity (at this stage). This effort is as futile now as trying to redeem a Dark Lord, say, Sauron. It could have been done before he became a Dark Lord, but now it's a pipe dream.

**TL;DR** - Do whatever you'd like, the world will probably turn to the worse sooner or later, but don't care about the global world. However, if you wanna progress further in evolution, make sure to do what you believe is the right/moral thing to do. Rise above materialism and mundane obsessions. Embody all that which you wish to see in others and live your life according to that. Wanna save humanity? Then improve yourself morally and intellectually. That's all. Changing others (or the entire world itself, for that matter) is not in our control.
